models/reviewDao.js

```diff
@@ -0,0 +1,117 @@
+const appDataSource = require("../util/orm");
+
+
+const searchUserToReview = async ( userId, productId ) => {
```

Comment:
dao 함수에 대한 변수 네이밍을 아래와 비슷하게 조금 더 명확하게 하면 좋을 것 같습니다.
```suggestion
const getReviewByUserId = async ( userId, productId ) => {
```